app/src/main/java/com/babylon/wallet/android/domain/model/resources/Resource.kt

```diff
@@ -168,7 +183,12 @@ sealed class Resource {
             get() = tagsMetadataItem?.tags?.map { Tag.Dynamic(name = it) }.orEmpty()
 
         val validatorAddress: String?
-            get() = validatorMetadataItem?.validatorAddress
+            get() = validatorMetadataItem?.address?.takeIf {
```

Comment:
It will be easier to share it once this PR is merged and handle this shared code in https://github.com/radixdlt/babylon-wallet-android/pull/677 PR's `validatorAddress()` method.